### DISCUSSION TITLE
region labels in egs_kerma

### DISCUSSION THREAD

**Discussion by nexustowers** [nrc-cnrc/EGSnrc#1231] [Questions] [answered]
Am I right in thinking region labels do not work in egs_kerma?

**Comment by rtownson** [comment, marked as answer]
That's correct, label support has yet to be added.

**Reply by nexustowers** [reply]
ok.  Thanks for replying

**Reply by rtownson** [reply]
Added in https://github.com/nrc-cnrc/EGSnrc/pull/1235
